src/main/java/net/fabricmc/fabric/api/client/model/ModelAppender.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2016, 2017, 2018 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.client.model;
+
+import net.minecraft.client.util.ModelIdentifier;
+import net.minecraft.resource.ResourceManager;
+
+import java.util.function.Consumer;
+
+@FunctionalInterface
+public interface ModelAppender {
+	void append(ResourceManager manager, Consumer<ModelIdentifier> modelAdder);
```

Comment:
-> void appendAll(ResourceManager manager, Consumer<ModelIdentifier> out); for clarity

Reply:
I'd say "appendAll" is a bit confusing, to me, and would rather go with "appendModels" or "appendAllModels" or "appendIds" or "appendAllIds" but I'll ask for a second opinion